packages/booter-lb3app/src/lb3app.booter.component.ts

```diff
@@ -0,0 +1,20 @@
+import {BootBindings, Bootstrapper} from '@loopback/boot';
```

Comment:
@raymondfeng what's the file naming convention here?

Reply:
Either `lb3app.booter.component.ts` or `lb3app-booter.component.ts` is fine with me.